MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvvmCross.Droid.Support.V7.AppCompat.csproj

```diff
@@ -138,6 +139,9 @@
       <Name>MvvmCross.Droid.Support.Core.UI</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <AndroidResource Include="Resources\Xml\Empty.xml" />
```

Comment:
Hm.  I'll try that out.  For reference I did file a bug (see the second commit message).